packages/@o3r/rules-engine/schematics/index.it.spec.ts

```diff
@@ -3,8 +3,7 @@
  * @jest-environment @o3r/test-helpers/jest-environment
  * @jest-environment-o3r-app-folder test-app-rules-engine
  */
-const o3rEnvironment = globalThis.o3rEnvironment;
```

Comment:
should be kept here

Reply:
Shy does it need to be dined before the import?

Reply:
to avoid the jsdoc being associated with the import
The const itself doesn't need to be first but the comment needs to be on the first line of the file for Jest to use it

Reply:
either we can disable the rule import/first for all our it.spec.ts (not in the recommended config) 
either we can set 
```
/* eslint-disable import/first -- for it test we want to have `o3rEnvironment` linked to the jsdoc of the jest environment setup */
```
before import

Reply:
https://github.com/AmadeusITGroup/otter/pull/2384